README.md

```diff
@@ -4,65 +4,68 @@ This application ingests SBOMs and Attestations into the Kusari Platform. It use
 
 ## Features
 
-- Upload single files or entire directories
-- OAuth2 client credentials authentication
-- Flexible configuration via CLI flags or environment variables
-- Supports ingestion of SBOMs and other documents
+-   Upload single files or entire directories
+-   OAuth2 client credentials authentication
+-   Flexible configuration via CLI flags or environment variables
+-   Supports ingestion of SBOMs and other documents
+-   Optional metadata tagging of uploaded documents
 
 ## Installation
 
 ```bash
 go get github.com/kusaridev/kusari-uploader
-```
 
 ## Usage
 
 ### Command-Line Flags
 
-```bash
 # Upload a single file
 ./kusari-uploader -f /path/to/file \
     -c CLIENT_ID \
     -s CLIENT_SECRET \
     -t TENANT_ENDPOINT \
-    -k TOKEN_ENDPOINT
+    -k TOKEN_ENDPOINT \
+    -a "package alias" \
+    -d "image"
 
 # Upload an entire directory
 ./kusari-uploader -f /path/to/directory \
     -c CLIENT_ID \
     -s CLIENT_SECRET \
     -t TENANT_ENDPOINT \
-    -k TOKEN_ENDPOINT
-```
-
-### Environment Variables
```

Comment:
I removed the Environment variable mapping as they are not working with Viper. I am not sure why....